spec/main_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+books = ['dummy'] # add finished books to this array
+
+RSpec.describe 'books' do
+  before(:all) do
+    books.each do |book|
+      # call the bake script for each book
+      output = `USE_LOCAL_KITCHEN=1 ./bake -b #{book} -i spec/books/#{book}/input.xhtml \
```

Comment:
Shouldn't use local kitchen, b/c that exists neither in production nor in GitHub Actions when we run these tests in PRs.

Reply:
The `./bake` and `spec/` assume the spec call is being run from the top-level directory.  In case it isn't, it'd be nice to make the paths still work.

`__dir__` will give you the directory of this file regardless of where it is run from.  So then `#{__dir__}/../bake` should always find the bake script.  Ditto for the input and output argument paths.

Reply:
I agree that on github, we shouldn't use the local version of kitchen. But developers are going to want to use specs to check that local changes to kitchen don't break previous books. Would it be better to pass this argument to the call to rspec?